target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -266,11 +266,6 @@
 
 	pinctrl: pinctrl {
 		compatible = "ralink,rt2880-pinmux";
-		pinctrl-names = "default";
-		pinctrl-0 = <&state_default>;
-
-		state_default: pinctrl0 {
-		};
```

Comment:
this assued `All pin groups function as gpio` but
if some pin groups function was changed by uboot
this `state_default ` may be needed ?

@arinc9

Reply:
All pins are marked as gpio under `ralink_pmx_group_enable()`. These will be written to the registry once the pinctrl driver kicks in after the bootloader.

https://github.com/torvalds/linux/blob/master/drivers/pinctrl/ralink/pinctrl-ralink.c#L117